packages/server/src/server.ts

```diff
@@ -14,7 +14,7 @@ import requestIp from 'request-ip';
 
 import { createApiMiddleware } from './middleware/api';
 import { createAuthenticateMiddleware } from './middleware/auth';
-import { cors } from './middleware/cors';
```

Comment:
removing this would break ability to use the sql version modules. Is there a way to do both, so we can temporarily get around the issue w/o breaking functionality?